chart/kubeapps/values.yaml

```diff
@@ -261,6 +261,14 @@ rbac:
   # Perform creation of RBAC resources
   create: true
 
+## Pod Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+##
+securityContext:
+  enabled: false
+  fsGroup: 1001
```

Comment:
I am not sure if in our case we need fsGroup since we do not have persistence

Reply:
Maybe we should comment these values out as examples?

Reply:
Even if there is no persistence the `fsGroup` is used to associate the volumes we use (in Nginx and tiller-proxy) but it's true that the daemons are not writing in those volumes so we can remove that if we want.

> Maybe we should comment these values out as examples?
 
I would comment those if after setting `enable: true` the parameters would be optional. In this case `runAsUser` is "mandatory" but we can comment out the `fsGroup`.